client/src/components/ReviewList.jsx

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react';
+import Review from './Review.jsx';
+
+const ReviewList = props => {
+  const [reviewCount, setReviewCount] = useState(0);
+  const [reviewsToShow, setReviewsToShow] = useState(0);
+  const [reviews, setReviews] = useState([]);
+  const [sortOption, setSortOption] = useState('relevant');
+
+  useEffect(() => {
+    // add ajax call for new reviewList
+  }, [sortOption]);
+
+  useEffect(() => {
+    setReviewCount(props.reviews ? props.reviews.length : 0);
+  }, [props.reviews]);
+
+
+  useEffect(() => {
+    setReviewsToShow(props.reviews ? reviewCount > 2 ? 2 : reviewCount : 0);
+    setReviews(props.reviews ? props.reviews.slice(0, reviewsToShow) : []);
+  }, [reviewCount]);
+
+  useEffect(() => {
+    setReviews(props.reviews ? props.reviews.slice(0, reviewsToShow) : []);
+  }, [reviewsToShow]);
```

Comment:
These `useEffect` and `useState` hooks seem promising. I might have to try them myself